k8s/staging/runners/public/windows/x86_64/v2/release.yaml

```diff
@@ -0,0 +1,185 @@
+                [[runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution]]
+                weight = 1
+                [runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term]
+                  topology_key = "topology.kubernetes.io/zone"
+                  [runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term.label_selector]
+                    [[runners.kubernetes.affinity.pod_affinity.preferred_during_scheduling_ignored_during_execution.pod_affinity_term.label_selector.match_expressions]]
+                      key = "spack.io/runner"
+                      operator = "In"
+                      values = ["true"]
+
+            [[runners.kubernetes.pod_spec]]
+              ttlSecondsAfterFinished = "300" # 5 minutes
+            [runners.kubernetes.pod_annotations]
+              "fluentbit.io/exclude" = "true"
+              "karpenter.sh/do-not-evict" = "true"
+              "gitlab/ci_pipeline_url" = "$CI_PIPELINE_URL"
+              "gitlab/ci_job_url" = "$CI_JOB_URL"
+              "gitlab/ci_project_url" = "$CI_PROJECT_URL"
+              "gitlab/ci_runner_description" = "$CI_RUNNER_DESCRIPTION"
+              "gitlab/ci_job_id" = "$CI_JOB_ID"
+            [runners.kubernetes.pod_labels]
+              "spack.io/runner" = "true"
+              "gitlab/ci_job_size" = "$CI_JOB_SIZE"
+              "metrics/gitlab_ci_pipeline_id" = "$CI_PIPELINE_ID"
+              "metrics/gitlab_ci_project_namespace" = "$CI_PROJECT_NAMESPACE"
+              "metrics/gitlab_ci_project_name" = "$CI_PROJECT_NAME"
+              "metrics/gitlab_ci_job_stage" = "$CI_JOB_STAGE"
+              "metrics/gitlab_ci_commit_ref_name" = "$CI_COMMIT_REF_NAME"
+              "metrics/spack_ci_stack_name" = "$SPACK_CI_STACK_NAME"
+              "metrics/spack_job_spec_pkg_name" = "$SPACK_JOB_SPEC_PKG_NAME"
+              "metrics/spack_spec_needs_rebuild" = "$SPACK_SPEC_NEEDS_REBUILD"
+            [runners.kubernetes.node_selector]
+              "kubernetes.io/os" = "windows"
+              # This is required to match the helper image based on this table
+              # https://gitlab.com/gitlab-org/gitlab-runner/-/blob/main/helpers/container/windows/version.go?ref_type=heads#L19-32
+              "node.kubernetes.io/windows-build" = "10.0.20348"
+            [runners.kubernetes.node_tolerations]
+              "spack.io/runner-taint=true" = "NoSchedule"
+              "windows=true" = "NoSchedule"
+
+      # default image
+      image: "mcr.microsoft.com/windows/servercore:ltsc2022"
+      imagePullPolicy: "if-not-present"
+      locked: false
+
+      tags: "spack,public,small,medium,win64,x86_64-win,x86_64_v2-win,aws"
+      runUntagged: false
+      secret: gitlab-gitlab-runner-secret # from gitlab release
+
+      cache: {}
+
+      services: {}
+
+      helpers: {}
+
```

Comment:
Just putting a comment here that if any of the suggestions are accepted from the other file, they should also be applied in this one.